propolis-server-lib/tests/artifacts.rs

```diff
@@ -0,0 +1,130 @@
+use anyhow::{bail, Result};
```

Comment:
This file is test-only, and allows thread-safe downloading of artifacts to be used in testing.